src/main/kotlin/no/nav/tilleggsstonader/sak/arbeidsfordeling/ArbeidsfordelingService.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.tilleggsstonader.sak.arbeidsfordeling
+
+import no.nav.tilleggsstonader.kontrakter.oppgave.Oppgavetype
+import no.nav.tilleggsstonader.sak.infrastruktur.config.getNullable
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.cache.CacheManager
+import org.springframework.stereotype.Component
+
+@Component
+class ArbeidsfordelingService(
+    private val arbeidsfordelingClient: ArbeidsfordelingClient,
+    @Qualifier("shortCache")
+    private val cacheManager: CacheManager,
+) {
+
+    companion object {
+        const val MASKINELL_JOURNALFOERENDE_ENHET = "9999"
+    }
+
+    fun hentNavEnhetId(ident: String, oppgavetype: Oppgavetype) = when (oppgavetype) {
+        Oppgavetype.VurderHenvendelse -> hentNavEnhetForOppfølging(ident, oppgavetype)?.enhetId
+        else -> hentNavEnhet(ident)?.enhetId
+    }
+
+    fun hentNavEnhet(ident: String): Arbeidsfordelingsenhet? {
```

Comment:
Kan ta det i egen PR, men her kommer vi ha behov for å senere bestille arbeidsfordeling på tema sammen med oppgavetype
https://favro.com/organization/98c34fb974ce445eac854de0/8a5276563da0f059dccd52a0?card=NAV-16320
Har en dialog med Hilde/oppgave om det